docs/source/usage/workflows/probeParticles.rst

```diff
@@ -62,23 +62,30 @@ and add it to ``VectorAllSpecies``:
 
 .. code-block:: cpp
 
-   CONST_VECTOR(
-       float_X,
-       3,
-       InCellOffset,
-       /* each x, y, z in-cell position component
-        * in range [0.0, 1.0) */
-       0.0,
-       0.0,
-       0.0
-   );
-   struct OnePositionParameter
-   {
-       static constexpr uint32_t numParticlesPerCell = 1u;
-       const InCellOffset_t inCellOffset;
-   };
-
-   using OnePosition = OnePositionImpl< OnePositionParameter >;
+    namespace startPosition
+    {
+        //! Configuration of initial in-cell particle position
+        struct OnePositionParameter
+        {
+            /** Maximum number of macro-particles per cell during density profile evaluation.
+             *
+             * Determines the weighting of a macro particle as well as the number of
+             * macro-particles which sample the evolution of the particle distribution
+             * function in phase space.
+             *
+             * unit: none
+             */
+            static constexpr uint32_t numParticlesPerCell = TYPICAL_PARTICLES_PER_CELL;
+
+            //! each x, y, z in-cell position component in range [0.0, 1.0)
+            static constexpr auto inCellOffset = float3_X(0., 0., 0.);
```

Comment:
`static constexpr auto inCellOffset = float3_X(0., 0., 0.).shrink<simSim>();`

I know the old example was for 3D only but we should help the users how to use it in 2D too.

Reply:
this should now work for both the `simDim=2D` and `simDim=3D`, the initial position is just truncated to the correct dimension